html2text/__init__.py

```diff
@@ -190,11 +190,9 @@ def previousIndex(self, attrs):
         """
         if "href" not in attrs:
             return None
-        i = -1
-        for a in self.a:
-            i += 1
-            match = False
 
+        match = False
+        for i, a in enumerate(self.a):
```

Comment:
Would be much better to add a comment here to justify the reason for using `enumerate`.

Maybe something like ?
```suggestion
        # Simplify getting previous Index by using enumerate()
        for i, a in enumerate(self.a):
```

Reply:
@jdufresne can we take care of this single suggestion to we can merge right away ?

Reply:
I guess I personally don't see what this comment adds. The [`enumerate`](https://docs.python.org/3/library/functions.html#enumerate) function is a builtin and well understood Python function such that commenting its purpose seems like noise to me.

Reply:
Makes sense to me as well.
Thanks ;)